argonaut/src/test/scala/org/http4s/argonaut/ArgonautSpec.scala

```diff
@@ -21,8 +21,9 @@ class ArgonautSpec extends JawnDecodeSupportSpec[Json] with Argonauts {
   "json encoder" should {
     val json = Json("test" -> jString("ArgonautSupport"))
 
-    "not have json content type" in {
-      jsonEncoder.headers.get(`Content-Type`) must_== None
+    "have json content type" in {
+      jsonEncoder.headers.get(`Content-Type`) must_== Some(
+        `Content-Type`(MediaType.application.json))
```

Comment:
Wow.  I'm trying to remember the justification for the old test.  Why hasn't it always been this way?